llvm/test/CodeGen/X86/vselect.ll

```diff
@@ -22,7 +22,7 @@ define <2 x i64> @masked_select_const(<2 x i64> %a, <2 x i64> %x, <2 x i64> %y)
 ; SSE41-LABEL: masked_select_const:
 ; SSE41:       # %bb.0:
 ; SSE41-NEXT:    movdqa %xmm0, %xmm3
-; SSE41-NEXT:    movdqa {{.*#+}} xmm4 = [4294967272,4294967272,4294967272,4294967272]
+; SSE41-NEXT:    pmovsxbd {{.*#+}} xmm4 = [4294967272,4294967272,4294967272,4294967272]
```

Comment:
This test already produces pmovsxbd before your patch.

Reply:
The base commit with the tests needs rebasing (and regenerating) on trunk first